docs/ops/normalization/BatchNormInference_1.md

```diff
@@ -4,39 +4,33 @@
 
 **Category**: *Normalization*
 
-**Short description**: *BatchNormInference* layer normalizes a `input` tensor by `mean` and `variance`, and applies a scale (`gamma`) to it, as well as an offset (`beta`).
+**Short description**: *BatchNormInference* performs Batch Normalizing Transform along a data batch. [Reference](https://arxiv.org/abs/1502.03167v2)
```

Comment:
Let's change to:
```
**Short description**: *BatchNormInference* performs Batch Normalization operation described in the [Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift](https://arxiv.org/abs/1502.03167v2) article.
```

I don't agree to use the word "Transform" because it is not a transformation but a mathematical operation.

Reply:
The reason I used this term to align with the way the paper names the algorithm. But it is true that causes confusion, so I have removed it.